css/style.css

```diff
@@ -0,0 +1,683 @@
+  line-height: 21px;
+  color: #000;
+  padding-top: 10px;
+}
+
+.h-6 {
+  font-size: 18px;
+  line-height: 5px;
+  color: #000;
+  padding-top: 7px;
+  font-weight: 700;
+}
+
+.h-6-1,
+.h-6-1-logo {
+  font-size: 18px;
+  line-height: 5px;
+  color: white;
+  font-weight: 700;
+}
+
+.h-6-1-logo {
+  display: flex;
+  align-items: flex-end;
+  height: 20px;
+  padding-right: 15px;
+}
+
+.h-6-logo {
+  font-size: calc(8px + (26 - 14) * ((100vw - 300px) / (1600 - 300)));
+  line-height: 5px;
+  color: #ef4430;
+  font-weight: 700;
+  padding-right: 2px;
+  padding-left: 2px;
+}
+
+.h-6-2 {
+  font-size: 16px;
+  line-height: 2px;
+  color: #ef4430;
+  font-weight: 900;
+}
+
+.h-6-4 {
+  font-size: 20px;
+  line-height: 2px;
+  color: #ef4430;
+  font-weight: 900;
+}
+
+.h-6-3 {
+  font-size: calc(6px + (26 - 14) * ((100vw - 300px) / (1600 - 300)));
+  line-height: 5px;
+  color: #000;
+  padding-top: 7px;
+  font-weight: 700;
+}
+
+.h-6-2:hover {
+  text-decoration: none;
+  color: #ef4430;
+}
+
+.h-5:hover,
+.h-6:hover {
+  color: #000;
+  text-decoration-color: rgb(238, 73, 87);
+}
+
+.h-7,
+.h-7-nav {
+  font-size: 15px;
+  font-weight: 700;
+  line-height: 12px;
+  color: #000;
+}
+
+.h-7-nav {
+  font-size: calc(10px + (26 - 14) * ((100vw - 300px) / (1600 - 300)));
+}
+
+.h-7:hover,
+.h-7-nav:hover {
+  color: #f72210;
+  text-decoration: none;
+}
+
+.h-8 {
+  font-size: 22px;
+  line-height: 27px;
+  color: #000;
+  padding-top: 7px;
+  font-weight: 700;
+}
+
+.last-con {
+  color: #383838;
+}
+
+.muted {
+  font-size: 12px;
+  line-height: 10px;
+  color: #55565a;
+  font-weight: 700;
+}
+
+.container-1 {
+  height: 40px;
+  position: relative;
+}
+
+.debate {
+  margin-left: 70%;
+  top: -20px;
+  left: 0;
+  position: absolute;
+}
+
+.vs-cont {
+  padding-top: 20px;
+  padding-bottom: 20px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.rounded-circle {
+  width: 70px;
+  height: 70px;
+}
+
+.vs {
+  margin-top: 35px;
+  margin-bottom: 35px;
+  height: 30px;
+  width: 30px;
+}
+
+.par {
+  font-size: calc(10px + (26 - 14) * ((100vw - 300px) / (1600 - 300)));
+}
+
+.card-center {
+  height: 13em;
+  display: flex;
+  justify-content: center;
+  align-items: flex-end;
+}
+
+.image-mag {
+  height: 18rem;
+  object-fit: cover;
+  background-size: cover;
+  background-repeat: no-repeat;
+  background-position: 50% 50%;
+}
+
+.image-mag-2 {
+  width: 100%;
+  object-fit: cover;
+  background-size: cover;
+  background-repeat: no-repeat;
+  background-position: 50% 50%;
+}
+
+.under-button {
+  display: flex;
+  background-color: #dbdcdd;
+  justify-content: center;
+  align-items: center;
+  height: 40px;
+}
+
+.under-button-1,
+.under-button-2 {
+  display: flex;
+  background-color: #ef4430;
+  width: 50%;
+  justify-content: center;
+  align-items: center;
+  height: 30px;
+}
+
+.under-button-logo {
+  display: flex;
+  background-color: white;
+  justify-content: center;
+  align-items: center;
```

Comment:
This is another opportunity to use bootstrap classes. ([Flex](https://getbootstrap.com/docs/4.3/utilities/flex/))